### ISSUE TITLE
OrientJS - Lexical error in batch

### ISSUE THREAD

**Issue by MarcelPitch** [orientechnologies/orientdb#5416]
Hi,

I'm using **orientDB 2.1.6** with **orientJS**.

When I try this simple batch : 

```javascript
var val = '";"';

gdb
	.let('test', function(s){

		s
			.create('vertex', 'V')
			.set({
				toto : val
			});
	})
	.commit()
	.all()
	.then(function(result){
		console.log(result);
	})
	.catch(function(err){
		console.log(err);
	})
	.done();
```

I have this error : 

```
[OrientDB.RequestError: Lexical error at line 1, column 31.  Encountered: <EOF> after : "\"\\\""]
  name: 'OrientDB.RequestError',
  message: 'Lexical error at line 1, column 31.  Encountered: <EOF> after : "\\"\\\\\\""',
  data: {},
  previous: [],
  id: 1,
  type: 'com.orientechnologies.orient.core.sql.parser.TokenMgrError',
  hasMore: 0
```

Thanks and have a nice day

**Comment by maggiolo00** [comment]
hi @MarcelPitch 
which version of orientjs?

**Comment by MarcelPitch** [comment]
Hi @maggiolo00,

orientjs is in **2.1.0**.

**Comment by MarcelPitch** [comment]
Hi @maggiolo00,

Do you have any news about this issue, or do you have any workaround ?

Have a nice day

**Comment by StarpTech** [comment]
@MarcelPitch thats the reason why I use my own transaction builder. The current implementation is for me not good. Try https://gist.github.com/StarpTech/ca0f923c49fd956c1c13

**Comment by MarcelPitch** [comment]
Hi @StarpTech,

Sorry for the delay.

OK, I've tested with your transaction builder, it works perfect ! Thanks a lot !

But the problem is that it makes me modify all the methods where an user can type '";'.

WDYT @maggiolo00, does this issue deserve a fix or not ?

Have a nice day, all

**Comment by maggiolo00** [comment]
yes
@MarcelPitch 

deserve a fix.

**Comment by MarcelPitch** [comment]
OK @maggiolo00, thx !

**Comment by maggiolo00** [comment]
hi @MarcelPitch 

it has been fix in 2.1.x by @luigidellaquila 
see here
https://github.com/orientechnologies/orientjs/issues/133
